source/software-architecture/software-architecture.rst

```diff
@@ -280,9 +282,15 @@ OS2iot REST API security
 
 In order to use the REST API exposed by OS2iot, the user must be authenticated.
 
-Authentication is done using the JWT gained from the :code:`/api/v1/auth/login` endpoint.
+There's two methods of authentication. The first method is done by using the JWT gained from the :code:`/api/v1/auth/login` endpoint.
```

Comment:
I think it's _There are_ instead of _There's_ because the noun _methods_ is plural :-)